packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -759,11 +754,6 @@ export class StorageService extends EventEmitter {
     })
   }
 
-  public resetCsrAndCertsValues() {
-    this.certificatesRequestsStore.resetCsrReplicatedMapAndId()
-    this.certificatesStore.resetValues()
-  }
```

Comment:
Is there any need to call `this.certificatesStore.resetValues()`?

Reply:
I think it's needed for restarting Quiet. We do not close backend completely when user leaves community, we just purge data and clean the state, reuse the tor process.

Reply:
I've added clean methods back in and added them to stores that didn't have them. So the code should be pretty similar to how it was before we refactors things (when everything was in the storage service).